app/settings/site/editor.directive.js

```diff
@@ -125,6 +125,8 @@ function (
                         let userLanguage = Session.getSessionDataEntry('language');
                         if ((userLanguage === undefined || userLanguage === null) && $scope.SystemLanguage !== newSystemLanguage) {
                             TranslationService.translate(newSystemLanguage);
+                            $scope.save = $translate.instant('app.save');
+                            $scope.saving = $translate.instant('app.saving');
                         }
```

Comment:
This + row 45-46 can be removed completely and changed to translation directly in the html-file :) 👍